src/core.c/Instant.pm6

```diff
@@ -50,7 +50,7 @@ my class Instant is Cool does Real {
         'Instant.from-posix(' ~ $posix.raku ~ ($flag ?? ',True)' !! ')')
     }
     method Bridge(Instant:   --> Num:D) { self.defined ?? self.tai.Bridge !! self.Real::Bridge }
-    method Num   (Instant:D: --> Num:D) { self.tai.Num    }
+    method Num   (Instant:D: --> Num:D) { nqp::div_n(nqp::decont_n($!tai.Num), 1000000000e0)   }
```

Comment:
Technically, this loses the subclassibility of `Instant`, because you access the attribute directly.  If we go this way, then there are other places where this could be done.  But maybe we shouldn't, and just use `self.tai` like everywhere else here?

Reply:
Then we can't do this optimization. `self.tai` is what causes the slowdown, since it returns a `Rat` and `.Num` on a `Rat` is slower than `.Num` on an `Int`.

Reply:
Ok, then add a method which would return `$!tai` as-is? It'd be even more correct to replace `$!tai` with `$!nano` and use that.

Reply:
Now implemented via `self.to-nanos`

Reply:
Hopefully, optimizer would eventually be capable of reducing double coercion into one.